README.md

```diff
@@ -0,0 +1,43 @@
+# Babelfish Tools
+
+[![Build Status](https://travis-ci.org/bblfsh/tools.svg?branch=master)](https://travis-ci.org/bblfsh/tools)
+[![codecov](https://codecov.io/gh/bblfsh/tools/branch/master/graph/badge.svg)](https://codecov.io/gh/bblfsh/tools)
+
+Language analysis tools on top of Babelfish
+
+## Build
+
+### With docker
+
+`make build`
+
+### Without docker
+
+`make build-internal`
```

Comment:
Now that we're using glide, we should probably add a `make depencencies` before `go build` as well.